Azure/azure-cli
# [AppService] Fix #13907: az webapp config ssl import: Change command to also import App Service Certificate and changing parameter names

Description
I am enabling az webapp config ssl import to also import an app service certificate to a webapp. At the moment, it just allows importing a KeyVault cert. With this change, it can be used for both of these. The breaking change here is changing the parameters, making key_vault optional, and setting up key_vault_certificate_name  to be deprecated. The new name parameter is going to be certificate, which could be a name, or an ID if the app service certificate is in a different subscription
Fixes #12427, #13907

Testing Guide
existing functionality is changed to:
az webapp config ssl import --resource-group rg --name webapp --certificate kvCertName --key-vault kvName
added functionality is:
az webapp config ssl import --resource-group rg --name webapp --certificate appServiceCertificateNamOrId

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).
